src/main/objects/RulesChecker.js

```diff
@@ -288,8 +288,43 @@ function checkConnection(prevChord, currentChord){
             }
         }
     }
-    if(prevChordFunctionTemp.functionName === Consts.FUNCTION_NAMES.SUBDOMINANT && currentChordFunctionTemp.functionName === Consts.FUNCTION_NAMES.DOMINANT){
-        //todo najbliższą drogą
+    if(prevChordFunctionTemp.functionName === Consts.FUNCTION_NAMES.SUBDOMINANT && currentChordFunctionTemp.functionName === Consts.FUNCTION_NAMES.DOMINANT
+        && prevChord.harmonicFunction.degree + 1 === currentChord.harmonicFunction.degree){
```

Comment:
Czy to jest zasada ogólna czy workaround, żeby sprawdzić czy jest połaczenie SIV -> DV?
Ogólnie to mam takie spostrzeżenie, iż strasznie skomplikowane są te zasady :(

Reply:
Dokładnie taka zasada jak w ifie :)